pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -206,7 +206,7 @@ func checkDiskSize(ctxPtr *zedmanagerContext) error {
 		if err != nil {
 			log.Errorf("checkDiskSize: err: %s", err.Error())
 			// Assume application is going down and its disks
-			// are going away.
+			// are going away. XXX which is utterly incorrect
```

Comment:
GetDiskSizeForAppInstance  can fail when error when "diskmetrics.GetDiskVirtualSize" fails..

Its just better to remove the comment here.. No point in retaining an incorrect comment.

There is a bigger question about accounting:

1) GetDiskSizeForAppInstance --> This is skipping RO images. Need to take that into account
        --> Should we count them separately - as separate from space take for App Instance?
        --> If the same AppImage is used for multiple App Instances - we definitely should not be double counting - hence the reason for skipping the check.
       --> This needs to be adjusted for the CAS world too - some layers can be shared across multiple applications

2) GetDiskSizeForAppInstance --> This is actually looking at the downloaded file and getting the Virtual size out of it - So the check will not actually work if the file is not downloaded.

Reply:
Checking the Size of the Drive we get from the controller.